Sources/Run/Routes/PostbackHandler.swift

```diff
@@ -155,15 +155,21 @@ extension Droplet {
         
         let botHostName = getBotHostName(config)
         
-        let url = "\(botHostName)/web?host=\(host)&user_id=\(subscriber.fb_messenger_id)&price=\(price)&product=\(product)"
+        let url = "\(botHostName)/web?host=\(host)&user_id=\(subscriber.fb_messenger_id)&price=\(price)&product=\(product)&event=1"
+        let url2 = "\(botHostName)/web?host=\(host)&user_id=\(subscriber.fb_messenger_id)&price=\(price)&product=\(product)&event=2"
         
-        let buttonClaimSpot = ["type": "web_url", "url": url, "messenger_extensions": "true", "title": "Claim your spot now"]
-        let pollResults = drop.carouselElement(title: "Join Hanna Lee show",
+        let buttonClaimSpot = ["type": "web_url", "url": url, "messenger_extensions": "true", "title": "Book the class"]
+        let buttonClaimSpot2 = ["type": "web_url", "url": url2, "messenger_extensions": "true", "title": "Book the class"]
+        let pollResults = drop.carouselElement(title: "Friday May 17th 7pm",
```

Comment:
p2: why pollResults? Do those link take them to booking or polling?

If booking, let's call it bookCardOne,  bookCardTwo

Reply:
Yep, I forgot to change it. Will fix it